Readme.md

```diff
@@ -207,6 +213,27 @@ associative array of currently supported SMTP authentication mechanisms
         related     // an array of attachments that you want to be related to the parent attachment
     }
 
+## new SMTP.SMTPConnection(options)
+
+	// options is an object with the following keys
+	options =
+	{
+		user 		// username for logging into smtp
+		password // password for logging into smtp
+		host		// smtp host
+		port		// smtp port (if null a standard port number will be used)
+		ssl		// boolean or object {key, ca, cert} (if true or object, ssl connection will be made)
+		tls		// boolean or object (if true or object, starttls will be initiated)
+		timeout	// max number of milliseconds to wait for smtp responses (defaults to 5000)
+		domain	// domain to greet smtp with (defaults to os.hostname)
+    authentication // array of preferred authentication methods ('PLAIN', 'LOGIN', 'CRAM-MD5', 'XOAUTH2')
+		logger // override the built-in logger (useful for e.g. Azure Function Apps, where console.log doesn't work)
+	}
+
+## SMTP.SMTPConnection#authentication
+
+associative array of currently supported SMTP authentication mechanisms
+
 ## Authors
 
 eleith
```

Comment:
@zackschuster please add your username too

Reply:
it's on L#240

Reply:
doh. great. mark as ready for review when you are ready to push this.